### PR TITLE
Add a data based relval test wf with lumi boundary crossing

### PR DESCRIPTION
**Pull request by fabiocos**
#### PR description:

New test wfs 136.8861 and 136.8862 (EGamma Run2018D) are proposed, expanding 136.886 with instructions to force a lumi boundary crossing within a short (100 events) run, and enabling ```nConcurrentlumis``` as discussed in #26348 . One test uses two lumi sections from different runs, while the other lumi sections from the same run. These tests are supposed to serve developments like those proposed by the Core FWK team in #26592.

#### PR validation:

The wf 136.8861 and 136.8862 run till the end.

### CODE REVIEW

File: Configuration/PyReleaseValidation/python/relval_steps.py [hunk 1]
```diff
@@ -478,6 +482,10 @@
 ### LS2 - MWGR ###
 steps['RunCosmics2019']={'INPUT':InputInfo(dataSet='/ExpressCosmics/Commissioning2019-Express-v1/FEVT',label='2019GR3',run=[330098],events=100000,location='STD')}
 
+#### Test of lumi section boundary crossing with run2 2018D ####
+Run2018Dml={320822: [[1,1]] , 320823: [[1,1]]}
+steps['RunEGamma2018Dml']={'INPUT':InputInfo(dataSet='/EGamma/Run2018D-v1/RAW',label='2018D',events=100000,location='STD', ls=Run2018Dml)}
```
Comment (makortel) [inline Configuration/PyReleaseValidation/python/relval_steps.py]: This leads to run boundary (between runs 320822 and 320823) in addition to lumi boundary, no?

Comment (fabiocos) [reply Configuration/PyReleaseValidation/python/relval_steps.py]: yes, I am working to a revised version that adds a lumi boundary only workflow

**Comment by fabiocos** [comment]
please test workflow 136.8861,136.8862

**Comment by chayanit** [comment]
+1

**Comment by kpedro88** [comment]
+upgrade

**Comment by fabiocos** [comment]
+1
